<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="grid_form">
     <item row="0" column="0">
      <widget class="QLabel" name="label_basic">
       <property name="text">
        <string>Basic:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="combo_basic">
       <item>
        <property name="text">
         <string>Warcaster</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Warjack</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Warlock</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Warbeast</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Solo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Grunt</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="hspacer_basic">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vspacer_buttons">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hbox_buttons">
     <item>
      <spacer name="hspacer_buttons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="push_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo_basic</tabstop>
  <tabstop>push_apply</tabstop>
  <tabstop>push_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
